plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js

```diff
@@ -49,6 +49,64 @@ jQuery( function ( $ ) {
 					'.wc_input_variations_price',
 					this.maybe_enable_button_to_add_price_to_variations
 				);
+
+			$( '.save-variations' ).on( 'click', function () {
+
+				$( '#variable_product_options' ).block( {
+					message: null,
+					overlayCSS: {
+						background: '#fff',
+						opacity: 0.6,
+					},
+				} );
+
+				const original_data = $(
+					'.woocommerce_variation_new_attribute_data'
+				).find( 'input, select, textarea' );
+
+				const data = {
+					post_id: woocommerce_admin_meta_boxes.post_id,
+					product_type: $( '#product-type' ).val(),
+					data: original_data.serialize(),
+					action: 'woocommerce_add_attributes_and_variations',
+					security:
+						woocommerce_admin_meta_boxes.add_attributes_and_variations,
+				};
+
+				$.post( woocommerce_admin_meta_boxes.ajax_url, data, function (
+					response
+				) {
+					if ( response.error ) {
+						// Error.
+						window.alert( response.error );
+						$( '#variable_product_options' ).unblock();
+					} else if ( response ) {
+						// Reload variations and attributes panel.
+						var this_page = window.location.toString();
```

Comment:
Nitpick: While I know we use the variable name (`this_page`) elsewhere, it's not very descriptive on its own. Would it be possible to change it to `this_page_url` or `page_url`?